odl/space/pspace.py

```diff
@@ -1081,6 +1081,32 @@ def ufuncs(self):
         """
         return ProductSpaceUfuncs(self)
 
+    @property
+    def real(self):
+        """Pointwise real part of the element."""
+        size = self.space.shape[0]
+        to_return = [None] * size
+        for (i, part) in zip(range(size), self.__parts):
+            to_return[i] = part.real
+        return self.space.element(to_return)
```

Comment:
This will be a problem since we want the real space, you need to implement `ProductSpace.real_space` and do
```python
return self.space.real_space.element(to_return)
```